features/primo/ping.feature

```diff
@@ -22,6 +22,18 @@ Feature: Primo is running
     Then my browser should respond with a success for Primo
     And my browser should resolve to BobCat
 
+  @production @warning
+  Scenario: Visiting BobCat NYUSH on production
+    Given I visit BobCat NYUSH
+    Then my browser should respond with a success for Primo
+    And my browser should resolve to BobCat
```

Comment:
@ericgriffis moreover we want to check that the browser resolves to BobCat NYUSH or NYUAD

`expect(page.find('.institution > span')).to have_content 'NYUSH'` etc, or whatever it looks like